### PR TITLE
chore: set cypress as optional dependencies

### LINKED ISSUE #101: Make cypress an optional dependency, make sure build works
## Description

With our current setup docker build downloads cypress which adds to build time.
Cypress is not required to build the image and it should be marked as optional

Moving cypress to `optionalDependencies` and running the following works:
```
pnpm install --frozen-lockfile --no-optional
```

but building fails because it also compiles the test files which cases the build to fail  
```
pnpm run build
```

## TODO

- [ ] Move cypress to `optionalDependencies`
- [ ] Ensure that build works

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #101

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,8 +56,9 @@
     "@nuxt/test-utils": "3.11.0",
     "@nuxtjs/i18n": "8.2.0",
     "@nuxtjs/sitemap": "5.1.2",
+    "@rollup/rollup-linux-arm64-gnu": "4.13.0",
```
Comment (kelsos) [inline package.json:59]: is there any reason why we need to specify this package? seeing that it is architecture/os specific?

Comment (lukicenturi) [reply package.json]: yeah I was just trying things.
So there is some issue with --no-optional and rollup
https://github.com/npm/cli/issues/4828 the answer is here

Comment (lukicenturi) [reply package.json]: updated

**Review by kelsos — APPROVED** [review]
lgtm thanks @lukicenturi

**Pull request merged**